app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -826,6 +826,17 @@ interface ProjectViewModel {
                     .compose(bindToLifecycle())
                     .subscribe { this.optimizely.track(CAMPAIGN_DETAILS_BUTTON_CLICKED, it.second, it.first.refTagFromIntent()) }
 
+            val isBackThisProjectCTA = this.pledgeActionButtonText
```

Comment:
I approve the naming here! On iOS we do something similar and we named it `shouldTrackCTATappedEvent`. Just in case you wanted to consider that.